src/main/java/pl/sda/jira/project/model/ProjectService.java

```
@@ -33,4 +33,10 @@ public void delete(long projectId) {
             throw new ProjectDoesntExistException();
         }
     }
+
+    public void update(Long projectId, String newProjectname) {
+        if(repository.isExist(projectId)){
+            repository.update(projectId, newProjectname);
+        }
```

should throw an exception when project does not exist